packages/next/build/utils.ts

```diff
@@ -1680,9 +1692,12 @@ server.listen(currentPort, (err) => {
   })
   handler = nextServer.getRequestHandler()
 
-  console.log("Listening on port", currentPort)
-})
-    `
+  console.log(
+    'Listening on port',
+    currentPort,
+    'url: http://localhost:' + currentPort
```

Comment:
Kept wanting to click on a link after server startup, so I added this similar to `next start`.